app/src/main/java/org/jellyfin/mobile/utils/SystemUtils.kt

```diff
@@ -130,3 +131,26 @@ fun Context.createMediaNotificationChannel(notificationManager: NotificationMana
         notificationManager.createNotificationChannel(notificationChannel)
     }
 }
+
+@Suppress("DEPRECATION")
+fun Context.getDownloadsPaths() = with(mutableListOf<String>()) {
```

Comment:
```suggestion
fun Context.getDownloadsPaths(): List<String> = mutableListOf<String>().apply {
```

`toList()` unnecessarily copies the List another time, just implicitly casting it to `List` is enough (and specifying the return type is generally recommended).